libpromises/evalfunction.c

```diff
@@ -9206,7 +9269,7 @@ static const FnCallArg HOSTSSEEN_ARGS[] =
 {
     {CF_VALRANGE, CF_DATA_TYPE_INT, "Horizon since last seen in hours"},
     {"lastseen,notseen", CF_DATA_TYPE_OPTION, "Complements for selection policy"},
-    {"name,address", CF_DATA_TYPE_OPTION, "Type of return value desired"},
+    {"name,address,key", CF_DATA_TYPE_OPTION, "Type of return value desired"},
```

Comment:
I think hostkey is more widely used.

```suggestion
    {"name,address,hostkey", CF_DATA_TYPE_OPTION, "Type of return value desired"},
```